packages/dd-trace/test/appsec/rasp/resources/shi-app/index.js

```diff
@@ -40,28 +40,42 @@ app.get('/shi/execSync/out-of-express-scope', async (req, res) => {
 })
 
 app.get('/cmdi/execFileSync', async (req, res) => {
-  childProcess.execFileSync('sh', ['-c', req.query.dir])
+  childProcess.execFileSync('sh', ['-c', req.query.command])
 
   res.end('OK')
 })
 
 app.get('/cmdi/execFileSync/out-of-express-scope', async (req, res) => {
   process.nextTick(() => {
-    childProcess.execFileSync('sh', ['-c', req.query.dir])
+    childProcess.execFileSync('sh', ['-c', req.query.command])
 
     res.end('OK')
   })
 })
 
 app.get('/cmdi/spawnSync', async (req, res) => {
-  childProcess.spawnSync('sh', ['-c', req.query.dir])
+  const result = childProcess.spawnSync(req.query.command)
+
+  if (result.error?.name === 'DatadogRaspAbortError') {
+    res.status(403).json({
+      error: 'DatadogRaspAbortError'
+    })
+    return
+  }
 
   res.end('OK')
 })
 
 app.get('/cmdi/spawnSync/out-of-express-scope', async (req, res) => {
```

Comment:
I think that this test doesn't make sense anymore. Because spawnSync is not throwing an exception, the error is not going to be catched by the global exception handler.